tests/pipelines/test_pipelines_visual_question_answering.py

```diff
@@ -109,6 +134,35 @@ def test_large_model_pt(self):
             [[{"score": 0.8799, "answer": "2"}, {"score": 0.296, "answer": "1"}]] * 2,
         )
 
+    @slow
+    @require_torch
+    def test_large_model_pt_git(self):
+        vqa_pipeline = pipeline("visual-question-answering", model="microsoft/git-base-textvqa")
+        image = "./tests/fixtures/tests_samples/COCO/000000039769.png"
+        question = "How many cats are there?"
+
+        outputs = vqa_pipeline(image=image, question=question)
+
+        self.assertEqual(
+            outputs,
+            [{"answer": "how many cats are there? 3"}],
```

Comment:
there are actually two cats in the picture, but the model returns 3, that's why 3 is used in this test.